yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -120,31 +122,21 @@ func parseSchemaFile(exportDir string) {
 		}
 	}
 
-	//TODO: convert below code into a for-loop
+	for objType, sqlStmts := range objSqlStmts {
+		if sqlStmts.Len() > 0 { // create .sql file only if there are DDLs
```

Comment:
Invert the condition to reduce the indentation level of the code:

```
for .... {
    if sqlStmts.Len() == 0 {
        continue
    }
    filePath := ...
    // Continue with further code.
}
```

Reply:
done